docs/experience-patterns/Issuance_pattern.md

```diff
@@ -87,7 +87,7 @@ Issuance UX design best practices (TBD)
 
 Implement and test the digital trust solution, conducting pilot trials to validate the approach and refine the implementation based on feedback.
 
-[Complete the Trial phase →](https://ditp-bc.atlassian.net/wiki/spaces/DTT/pages/204865619)
+[Complete the Trial phase →](https://bcgov.github.io/digital-trust-toolkit/docs/delivery-manual/trial/)
 
 [Becoming an issuer→](https://ditp-bc.atlassian.net/wiki/spaces/DTT/pages/207126646/Becoming+an+Issuer)
```

Comment:
Maybe remove the link for now as it leads to Confluence and add a note that it will be added soon